tags/tag-charters/network-charter.md

```diff
@@ -88,23 +80,5 @@ Clarify and further concepts of cloud native networking, traffic management and
 
 # Governance &amp; Operations
 
-This TAG follows the [standard operating model](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#operating-model) provided by the TOC unless otherwise stated here.
-
-# Operating Model
-
-## Chairs:
-
-- TOC Liaison: Duffie Cooley, Ricardo Rocha
-- TAG Chairs: [Lee Calcote](https://twitter.com/lcalcote), [Ken Owens](https://twitter.com/kenowens12)
-
-In accordance with the [elections and terms](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections) follow the [CNCF TAG definition](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections).
-
-## Communications
+This TAG follows the [standard operating model](https://github.com/cncf/toc/blob/main/tags#operating-model) provided by the TOC unless otherwise stated here.
 
-- Slack Channel ([#tag-network](https://app.slack.com/client/T08PSQ7BQ/CMG237Z5Z))
-- Join [TAG-Network](mailto:tag-network@lists.cncf.io) mailer at [lists.cncf.io](https://lists.cncf.io)
-- Repo: [https://github.com/cncf/tag-network](https://github.com/cncf/tag-network)
-- Meetings: 
-  - Time: Network TAG meets the 1st and 3rd Thursday of every month at 11am Pacific (here's [a link](https://goo.gl/eyutah) to a public Google calendar that you can subscribe to).
-  - Location: Zoom - [https://zoom.us/my/cncftagnetwork](https://zoom.us/my/cncftagnetwork)
-  - [Meeting Minutes](https://docs.google.com/document/d/18hYemFKK_PC_KbT_TDBUgb0rknOuIhikkRxer4_bv4Q/edit#)
```

Comment:
Hi @ZackButcher, according to my understanding from the templates for both the charter, and the readme; the **Communcations** link should exist only in the README file.

If you however feel strongly otherwise, then you need only say the word and I would make the adjustments accordingly.

Reply:
Correct. For standardization the TOC want to keep meetings and communications in the TAG's Home README

Reply:
Sounds good, thanks!